### PR TITLE
Add release and cleanup actions

### PR DESCRIPTION
**Pull request by george-zubrienko**

### CODE REVIEW

File: .github/workflows/registry-cleanup.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    timeout-minutes: 10 # stop the task if it takes longer
+
+    steps:
+      - name: Delete old package versions of nexus-configuration-controller
+        uses: actions/delete-package-versions@v5.0.0
+        with:
+          package-name: 'nexus-configuration-controller'
+          package-type: container
+          token: ${{ secrets.GITHUB_TOKEN }}
+          min-versions-to-keep: 10
```
Comment (s-vitaliy) [inline .github/workflows/registry-cleanup.yaml:25]:
```suggestion
          min-versions-to-keep: 100
```

File: .github/workflows/registry-cleanup.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          delete-only-pre-release-versions: "true"
+
+      - name: Delete old package versions of helm/nexus-configuration-controller
+        uses: actions/delete-package-versions@v5.0.0
+        with:
+          package-name: 'helm/nexus-configuration-controller'
+          package-type: container
+          token: ${{ secrets.GITHUB_TOKEN }}
+          min-versions-to-keep: 10
```
Comment (s-vitaliy) [inline .github/workflows/registry-cleanup.yaml:34]:
```suggestion
          min-versions-to-keep: 100
```

File: .github/workflows/registry-cleanup.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    steps:
+      - name: Delete old package versions of nexus-configuration-controller
+        uses: actions/delete-package-versions@v5.0.0
+        with:
+          package-name: 'nexus-configuration-controller'
+          package-type: container
+          token: ${{ secrets.GITHUB_TOKEN }}
+          min-versions-to-keep: 10
+          delete-only-pre-release-versions: "true"
```
Comment (s-vitaliy) [inline .github/workflows/registry-cleanup.yaml:26]: This argument behaves weirdly, I would suggest to use a regex to limit versions affected by the pipeline: https://github.com/SneaksAndData/arcane-stream-sqlserver-change-tracking/blob/e6a34f331ced1854088ec3968dcf8effcefe2974/.github/workflows/cleanup-repository.yaml#L26

**Review by s-vitaliy — COMMENTED** [review]
Also we may need the push to ECR action: https://github.com/SneaksAndData/arcane-stream-sqlserver-change-tracking/blob/e6a34f331ced1854088ec3968dcf8effcefe2974/.github/workflows/publish-ecr-public.yaml

**Review by s-vitaliy — APPROVED** [review]

**Comment by george-zubrienko** [comment]
> Also we may need the push to ECR action: https://github.com/SneaksAndData/arcane-stream-sqlserver-change-tracking/blob/e6a34f331ced1854088ec3968dcf8effcefe2974/.github/workflows/publish-ecr-public.yaml

Will add in a separate PR

**Pull request merged**